### ISSUE TITLE
DND not enabled during slack screensharing sessions - Big Sur

### ISSUE THREAD

**Issue by veics** [bdkjones/MuzzleApp#96]
when starting a screen share in Slack, DND is not enabled, but it is enabling it after stopping the screenshare.

**Comment by Moulick** [comment]
Same happens with MS Teams

**Comment by borna761** [comment]
Same with Zoom

**Comment by veics** [comment]
now it's not working at all anymore. No matter what application you use

**Comment by veics** [comment]
updates, I am on "ventura". The DND is now working, but after stop sharing the screen, the DND is not turned off, but it remains like this :).
For the moment, I will start using the out of the box functionality from mac for stopping the notifications while I'm sharing the screen.

**Comment by Moulick** [comment]
Yep, same here. DND is not disabled after exiting a screen share. This has led to me missing messages often.
